### PR TITLE
Ac and Dc variant UR arm support for ROX 

### PR DESCRIPTION
**Pull request by AdarshKaran**
Cherry picked the necessary changes for adding ac and dc variant option and modified arm position to fit ac cabinet version, modified README

AC variant cabinet H2 has to be updated from 790 to 706mm in the documentation 
[ROX](https://neobotix-docs.de/hardware/en/manipulators/emrox/mechanical.html)

### CODE REVIEW

File: rox_bringup/configs/ur_config/ompl_planning.yaml
Comment (padhupradheep) [inline rox_bringup/configs/ur_config/ompl_planning.yaml:1]: Why do we need this file?

File: rox_bringup/configs/ur_config/ur10/moveit_controllers.yaml
Comment (padhupradheep) [inline rox_bringup/configs/ur_config/ur10/moveit_controllers.yaml:1]: Why is this needed?

File: rox_bringup/configs/ur_config/ur10/ur_controllers.yaml
Comment (padhupradheep) [inline rox_bringup/configs/ur_config/ur10/ur_controllers.yaml:1]: Can't we just have a single file and add prefix for the UR variant that we want to use?

File: rox_navigation/launch/navigation.launch.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def generate_launch_description():
         )
 
     declare_use_sim_time_cmd = DeclareLaunchArgument(
-            'use_sim_time', default_value='false',
+            'use_sim_time', default_value='true',
```
Comment (padhupradheep) [inline rox_navigation/launch/navigation.launch.py]: default should be false.

**Review by padhupradheep — CHANGES_REQUESTED** [review]
Overall LGTM! There are bits and pieces that needs to be changed.

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+    'use_ur_dc':
+        Set this argument to True if you have an UR arm with DC variant
+        (default: 'false')
+```
+
+## Simulating the rox Robot with a UR Arm
+
+To simulate the rox robot with a Universal Robotics (UR) arm, you need to set the `arm_type` parameter. Below is an example of how to launch the simulation with the `ur10` arm type
+
+`ros2 launch rox_bringup bringup_sim_launch.py arm_type:=ur10`
```
Comment (padhupradheep) [inline README.md]: These are just hints. Could you link into to our documentation website as well?

P.S.: There is a huge update of docs coming soon.

**Comment by padhupradheep** [comment]
Once you have done those changes requested, we will fix those merge conflicts together, during our next meeting. 

But overall a good job. :clap: 

CC: @neoflv @neomob

File: rox_bringup/configs/ur_config/ompl_planning.yaml
Comment (AdarshKaran) [reply rox_bringup/configs/ur_config/ompl_planning.yaml:1]: i have now removed it since this package does not need any moveit configs

File: rox_bringup/configs/ur_config/ur10/moveit_controllers.yaml
Comment (AdarshKaran) [reply rox_bringup/configs/ur_config/ur10/moveit_controllers.yaml:1]: i have also removed this

File: rox_bringup/configs/ur_config/ur10/ur_controllers.yaml
Comment (AdarshKaran) [reply rox_bringup/configs/ur_config/ur10/ur_controllers.yaml:1]: I tried to do it this way but the current implementation of "simulation_controllers" parameter in UR description package accepts only a file path. But when substituting dynamic parameters during runtime and safe loading the yaml, a dictionary is returned which cannot be passed here. hence instead of creating a temporary file during runtime i have kept this as such Pradheep.

File: rox_navigation/launch/navigation.launch.py (hunk 1 above)
Comment (AdarshKaran) [reply rox_navigation/launch/navigation.launch.py]: I have changed this also Pradheep

File: README.md (hunk 2 above)
Comment (AdarshKaran) [reply README.md]: I have added the link to the docs, specifically [Docs](https://neobotix-docs.de/ros/ros2/simulation_modern.html#rox)

File: rox_bringup/configs/ur_config/ompl_planning.yaml
Comment (padhupradheep) [reply rox_bringup/configs/ur_config/ompl_planning.yaml:1]: why is it still there?
